### PR TITLE
on/convert-docs

### PR DESCRIPTION
**Pull request by oshi98**
- [x] Restructure assets folder (follow Docs team conventions)
- [x] Modify feedback feature to direct to GitHub issues
- [x] Convert md files

### CODE REVIEW

**Review by zainhaq-h2o — APPROVED** [review]
@oshi98 I fixed a broken link on the index page with https://github.com/h2oai/h2o-health/pull/13/commits/71c5b3ea0d70153ebe5771b628820b9765cb07e1
LGTM otherwise, thanks!

File: README.md [hunk 1]
```diff
@@ -1,18 +1,62 @@
...
+1. Run: `make run-doc`
+
+    - **Documentation**: To view your edits for the documentation (located at `docs/docs`), navigate to the provided URL (for example, `http://localhost:3000/h2o-health/`). 
+        - **Debug** 
+            - If you do not see anything (that is, a white screen) after clicking **Next** , run the following command and try again: `make setup-doc` 
+
+
+
+
+
```
Comment (sherenem) [inline README.md:62]: I think there's some more content that should go into the readme according to https://github.com/h2oai/docusaurus-template/pull/23/files

File: docs/docs/team.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
-# :material-account-supervisor-circle-outline: Team 
+import {interpolate} from '@docusaurus/Interpolate';
+const feedback = interpolate('https://github.com/h2oai/docs-issues-requests/issues/new?assignees=5675sp&labels=area%2Fdocumentation%2C+h2o-health%2C+haic-app&body=%23%23%23%20Documentation%20issue%2Frequest%0A%0A%3C!--%20Please%20provide%20a%20clear%20and%20concise%20description%20of%20the%20documentation%20issue%2Frequest%20--%3E%0A%0A%23%23%23%20Additional%20context%0A%0A%3C!--%20Please%20add%20any%20other%20context%20about%20the%20issue%2Frequest%20here%20(e.g.%2C%20images)%20--%3E%0A%0A%23%23%23%20Page%20details%20%0A%0A-%20Application%20name%3A%20H2O%20Health%0A-%20Application%20version%3A%20{version}%0A-%20Page%20title%3A%20{title}%20&title=%5BHAIC-APP%5D', {title: metadata.permalink, version: metadata.version});
+
+# Team
```
Comment (sherenem) [inline docs/docs/team.md:4]: @5675sp @oshi98 are we adding this "Team" page in all docs or is it something specific to H2O Health?

File: docs/docs/gene-mutation/predictions-workflow.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import {interpolate} from '@docusaurus/Interpolate';
+const feedback = interpolate('https://github.com/h2oai/docs-issues-requests/issues/new?assignees=5675sp&labels=area%2Fdocumentation%2C+h2o-health%2C+haic-app&body=%23%23%23%20Documentation%20issue%2Frequest%0A%0A%3C!--%20Please%20provide%20a%20clear%20and%20concise%20description%20of%20the%20documentation%20issue%2Frequest%20--%3E%0A%0A%23%23%23%20Additional%20context%0A%0A%3C!--%20Please%20add%20any%20other%20context%20about%20the%20issue%2Frequest%20here%20(e.g.%2C%20images)%20--%3E%0A%0A%23%23%23%20Page%20details%20%0A%0A-%20Application%20name%3A%20H2O%20Health%0A-%20Application%20version%3A%20{version}%0A-%20Page%20title%3A%20{title}%20&title=%5BHAIC-APP%5D', {title: metadata.permalink, version: metadata.version});
+
+# Predictions Workflow
```
Comment (sherenem) [inline docs/docs/gene-mutation/predictions-workflow.md:4]:
```suggestion
# Predictions workflow 
```

File: docs/docs/gene-mutation/predictions-workflow.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+- To learn about how Snowflake supports Gene Mutation AI, see [Snowflake](snowflake/snowflake.md).
+- To learn about datasets in Gene Mutation AI, see [Datasets](datasets/datasets.md).
+
+## Step 2: Run Predictions
+
+To run predictions *Gene Mutation AI* requires particular models built in H2O Driverless AI. 
+
+- To learn about all the H2O Driverless AI models supported within *Gene Mutation AI*, see [Predictions](predictions/predictions.md).
+
+## Step 3: View Results
```
Comment (sherenem) [inline docs/docs/gene-mutation/predictions-workflow.md:27]: Sentence case these headings

**Comment by sherenem** [comment]
The "gene-mutation/assets" folder only has images and no related `.md` file. Is this intentional?

File: docs/docs/team.md (hunk 2 above)
Comment (oshi98) [reply docs/docs/team.md:4]: Not in all docs but in H2O Health, there were [Team pages](https://docs.h2o.ai/h2o-health/team/) with developers' names. Maybe we can check with the dev team if it's necessary. If not, we can remove it from the new docs. WDYT?

**Comment by oshi98** [comment]
> Thanks @oshi98. I left a few suggestions. Also, the "gene-mutation/assets" folder only has images and no related `.md` file. Is this intentional?

Oh, yes, I'll remove it. Thanks for pointing it out.

File: README.md (hunk 1 above)
Comment (oshi98) [reply README.md:77]: Added the missing content - https://github.com/h2oai/h2o-health/pull/13/commits/7573e2ca85859b276156bc54c02bc3e81c05313e

File: docs/docs/gene-mutation/predictions-workflow.md (hunk 4 above)
Comment (oshi98) [reply docs/docs/gene-mutation/predictions-workflow.md]: Fixed https://github.com/h2oai/h2o-health/pull/13/commits/f63c8dcf6aab9f76d6ad29525f61133d9f44d785

File: docs/docs/team.md (hunk 2 above)
Comment (5675sp) [reply docs/docs/team.md:4]: its specific to H2O Health @sherenem

File: docs/docs/index.md [hunk 5]
```diff
@@ -23,24 +25,16 @@ At [H2O Health](https://h2o.ai/solutions/industry/health/), we build AI apps tha
...
+<div class="home-suite-container">
+  <a href="epidemiology/covid-19-hospital-occupancy-simulator/overview" class="home-app-link">
+    <img src={chos} alt="" width="150"/> <br/>
+    <span>Covid-19 Hospital Occupancy Simulator (CHOS)</span>
+  </a>
+  <br/>
+  <a href="gene-mutation/overview" class="home-app-link">
+    <img src={geneMutation} alt="" width="100"/> <br/>
+    <span>Gene Mutation AI</span>
+  </a>
```
Comment (5675sp) [inline docs/docs/index.md:37]: @oshi98 Please position the logos in a horizontal way as its currently structure here: https://docs.h2o.ai/h2o-health/, also, please add the navigation section under this line

**Comment by 5675sp** [comment]
Hello @oshi98,

Please remove the extra spaces between lines in this files: 

- http://localhost:3000/h2o-health/gene-mutation/snowflake
- http://localhost:3000/h2o-health/gene-mutation/datasets
- http://localhost:3000/h2o-health/gene-mutation/predictions
- http://localhost:3000/h2o-health/gene-mutation/team

Please check all pages, thanks 


Also, for small images please center them, please use the below syntax, for example: 

```
<div style={{textAlign: 'center'}}>

![go-to-instance.png](go-to-instance.png)

</div>
```

**Comment by 5675sp** [comment]
@oshi98 as you review and convert please keep in mind a sentence case format

**Review by 5675sp — APPROVED** [review]

**Pull request merged**